catalyst/dl/callbacks/mixup.py

```diff
@@ -0,0 +1,45 @@
+from typing import List
+from catalyst.dl.core.state import RunnerState
+from catalyst.dl.callbacks import CriterionCallback
+import numpy as np
+import torch
```

Comment:
to follow lib code standarts, its better to use the following import sequence:
- core python libs (typing, collection, etc)
- python libs (numpy, etc)
- dl libs (pytorch, etc)
- catalyst imports

Reply:
Done, thanks.